custom/opm/reports.py

```diff
@@ -1043,7 +1043,7 @@ def headers(self):
 
     def get_matching_awcs(self):
         filter_on = 'awc' if self.awcs else 'gp' if self.gp else 'block'
-        for block, gps in get_hierarchy().items():
+        for block, gps in user_data_as_hierarchy().items():
```

Comment:
This uses awc doc_id instead of name.  I think for this switchover to work, you'd need to also update the filter values to use that as the key rather than awc_code, right?